### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by minthe**

### CODE REVIEW

**Comment by minthe** [comment]
@mariekart42 i needed to make some changes in your codebase. lets review them together

File: ms_game_chat/backend/Dockerfile [hunk 1]
```diff
@@ -10,4 +10,4 @@ RUN pip install --no-cache-dir -r requirements.txt
 WORKDIR /code
 COPY . .
 
-CMD ["sh", "-c", "python manage.py makemigrations; python manage.py migrate; python manage.py runserver 0.0.0.0:6969"]
\ No newline at end of file
+CMD ["./ms_game_chat_start.sh"]
```
Comment (minthe) [inline ms_game_chat/backend/Dockerfile:13]: its more reliable as script than  chaining commands in CMD

File: ms_game_chat/backend/backend_app/views.py [hunk 2]
```diff
@@ -11,19 +11,19 @@
...
     return render(request, 'goToFrontend.html')
 
-
 # Create new user:
 # - Endpoint: game/user/{user_id}/
 # - Method:   POST
 # - Payload:  username:string, avatar:string
 @require_POST
 @jwt.token_required
-def createUser(request, user_id):
```
Comment (minthe) [inline ms_game_chat/backend/backend_app/views.py:21]: @mariekart42 updated your function for the new endpoint layout

File: ms_game_chat/backend/requirements.txt [hunk 3]
```diff
@@ -3,5 +3,4 @@ psycopg[binary]
 channels~=4.0
 daphne~=4.0
 channels_redis~=4.1
-ft_jwt
-
+ft_jwt==0.1.0
```
Comment (minthe) [inline ms_game_chat/backend/requirements.txt:6]: @mariekart42 without version number we did not get the newest version, had to hardcode now a specific release

**Review by mariekart42 — APPROVED** [review]

**Pull request merged**